kedro/templates/project/{{ cookiecutter.repo_name }}/src/tests/test_run.py

```diff
@@ -11,13 +11,24 @@
 from pathlib import Path
 
 import pytest
+
+from kedro.config import ConfigLoader
 from kedro.framework.context import KedroContext
+from kedro.framework.hooks import _create_hook_manager
+
+
+@pytest.fixture
+def config_loader():
+    return ConfigLoader(conf_source=str(Path.cwd()))
```

Comment:
I think should be 
```
str(Path.cwd() / settings.CONF_SOURCE)
```
... but do check that gives the right thing.

Reply:
For the test it doesn't really matter, but I guess it would be useful to add this in docs so users know how to configure their ConfigLoader properly. The starters were already changed and so was the "empty" project template. But all with `ConfigLoader(conf_source=str(Path.cwd()))`

Reply:
Actually on second thought, I think it should just be `settings.CONF_SOURCE`. The example we give in the docs is: 

```
from kedro.config import ConfigLoader

conf_loader = ConfigLoader(conf_source="conf", env="local")
```

so for clarity I'll change it all to take the conf source value from settings.

Reply:
I think the docs are probably wrong here. Here's what we do in `session._get_config_loader`:
```
    def _get_config_loader(self) -> ConfigLoader:
        """An instance of the config loader."""
        env = self.store.get("env")
        extra_params = self.store.get("extra_params")

        config_loader_class = settings.CONFIG_LOADER_CLASS
        return config_loader_class(
            conf_source=str(self._project_path / settings.CONF_SOURCE),
            env=env,
            runtime_params=extra_params,
            **settings.CONFIG_LOADER_ARGS,
        )
```

Reply:
So all the docs here are wrong then? https://kedro.readthedocs.io/en/latest/kedro_project_setup/configuration.html 
Is this still possible?
```
conf_paths = ["conf/base", "conf/local"]
conf_loader = ConfigLoader(conf_paths)
```

Reply:
Yeah, the new signature of `ConfigLoader` is this:
```
    def __init__(
        self,
        conf_source: str,
        env: str = None,
        runtime_params: Dict[str, Any] = None,
        *,
        base_env: str = "base",
        default_run_env: str = "local",
    ):
```

Reply:
@AntonyMilneQB I've reverted my earlier change so now this PR is just for adding the right arguments for the `test_run` files. I'll do a follow up PR to change any other reference to `ConfigLoader`